CHANGELOG.md

```diff
@@ -62,6 +62,7 @@ The types of changes are:
 * Removed pypi badge pointing to wrong package [#452](https://github.com/ethyca/fidesops/pull/452)
 * Audit imports and references [#479](https://github.com/ethyca/fidesops/pull/479)
 * Switch to using update method on PUT permission endpoint [#500](https://github.com/ethyca/fidesops/pull/500)
+* Fixed type errors for privacy center build [#540](https://github.com/ethyca/fidesops/pull/540)
```

Comment:
It looks like the changelog file is a little behind what is in main. Git can probably merge it safely but I'm not sure if it will place this update in the correct place now that there are multiple `fidesops` versions in the changelog file within main. 

Could main be merged into this branch to ensure that the update goings in the unreleased section like we expect?